cdc-mysql/cdc.sh

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
+
+set -x
+
+readonly PROGNAME=$(basename "${0}")
+readonly PROGDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+readonly ARGS=("$@")
+
+[[ -x "$(command -v "mysql")" ]] || { echo "Missing mysql"; exit; }
+[[ -x "$(command -v "docker")" ]] || { echo "Missing docker"; exit; }
+[[ -x "$(command -v "fluvio")" ]] || { echo "Missing fluvio"; exit; }
+
+pkill_producer_consumer() {
+  pkill cdc-producer
+  pkill cdc-consumer
+}
+
+fluvio_uninstall() {
+  fluvio cluster uninstall --local
+}
+
+docker_kill_mysql() {
+  docker kill mysql-consumer mysql-producer
+}
+
+docker_rm_mysql() {
+  docker rm mysql-consumer mysql-producer
+}
+
+docker_rmi_mysql() {
+  docker rmi mysql-80
+}
+
+clear_mysql_volume() {
+  # Use first arg for mysql volume mount, or ~/mysql-cdc as default
+  local path="${1:-"${HOME}/mysql-cdc"}"
+
+  sudo rm -rf "${path}"
+}
+
+fluvio_install() {
+  fluvio cluster install --local
+}
+
+# shellcheck disable=SC2120
+fluvio_topic() {
+  local topic="${1:-"rust-mysql-cdc"}"; shift
+  fluvio topic create "${topic}"
+  sleep 1
+}
+
+docker_mysql() {
+  local name=${1:-"mysql-producer"}; shift
+  local path=${1:-"${HOME}/mysql-cdc/cdc-producer"}; shift
+  local port=${1:-"3080"}; shift
+  local image="mysql-80"
+
+  mkdir -p "${path}"
+  docker build "${PROGDIR}/docker" -t "${image}"
+
+  docker run -p "${port}:3306" \
+    -v "${path}:/var/lib/mysql" \
+    -v "scripts:/docker-entrypoint-initdb.d/" \
+    --name "${name}" \
+    -e "MYSQL_ROOT_PASSWORD=root" \
+    -d "${image}" \
+    --server-id=1 \
+    --log-bin=/var/lib/mysql/binlog.index \
+    --binlog-format=row \
+    --default-authentication-plugin=mysql_native_password
+}
+
+reset() {
+  # Use first arg for mysql volume mount, or ~/mysql-cdc as default
+  local mysql_volume_mount=${1:-"${HOME}/mysql-cdc"}; shift
```

Comment:
I sort of agree with you, except for that the CDC example itself isn't really fluvio proper. I wonder if we could make a `.fluvio-cdc` directory or something like that.

Reply:
sure. as long as indicating that is fluvio demo otherwise, you will wonder where data is coming from.   Also, it might be better if this is written to using `/tmp`